src/dropdown/list/_dropdown-list.scss

```
@@ -2,15 +2,53 @@
 @import "../../ui/reference/_measurement.scss";
 
 .dropdown-list {
+  position: absolute;
+  z-index: 1;
+
   width: 100%;
+  max-height: 320px;
 
   box-sizing: border-box;
+  overflow: auto;
 
   padding: 0;
 
   background-color: white;
   border: 1px solid var(--default-border-color);
   border-radius: var(--small-radius-size);
+  visibility: hidden;
+  opacity: 0;
+
+  &.top {
+    top: 0;
+
+    transform: translateY(-100%);
+  }
+
+  &.bottom {
+    bottom: -15px;
+
+    transform: translateY(100%);
+  }
+
+  &.left {
+    top: 0;
+    left: 0;
+
+    transform: translate(-100%, -50%);
+  }
+
+  &.right {
+    top: 0;
+    right: 0;
+
+    transform: translate(100%, -50%);
+  }
+}
+
+.dropdown-list__is-visible {
```

```suggestion
.dropdown-list--is-visible {
```
Shouldn't we separate `is-visible` with `--` because it represents the state of the element (i.e. a modifier)?

---

Yep, thanks for catching this!